ios/RNSpokestack.swift

```diff
@@ -164,14 +164,21 @@ class RNSpokestack: RCTEventEmitter, SpokestackDelegate {
 
     func classification(result: NLUResult) {
         if let resolve = resolvers.removeValue(forKey: RNSpokestackPromise.classify) {
+            var slots: [String:[String:String]] = [:]
+            if let resultSlots = result.slots {
+                for (name, slot) in resultSlots {
+                    slots[name] = [
+                        "type": slot.type,
+                        "value": RCTConvert.nsString(slot.value),
```

Comment:
Doesn't this mean that `value` will always be a string for iOS devices, but could be a different type on Android (given the `when` in the Kotlin)?

Reply:
both platforms will end up being a string in TypeScript anyway due to the react native bridging, so losing the type here won't matter.

Reply:
See [here](https://github.com/spokestack/react-native-spokestack/pull/91/files#diff-e14afd66f0fd5d3c34efc5df4a97dad6f7e33d949834498e299d4459809096b7R38-R43) for my Kotlin reference point. This maintains the native typing to some degree.

Reply:
Actually, the types can be maintained across the bridge. I'll see if I can figure out an equivalent to the Android type checking.

Reply:
This should work. https://github.com/spokestack/react-native-spokestack/pull/91/files#diff-79b0b52fee2f77d6112afd3e3d1e53f4d5dd6e0bd7b9b515f87235652e310d02R171